### PR TITLE
Switch to postgres

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala [hunk 1]
```diff
@@ -21,58 +20,67 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers {
...
 
-    fileService.addFile(AddFilesInput(consignmentId, 1),Some(userId)).await()
+    fileService.addFile(AddFilesInput(consignmentUuid, 1),Some(userId)).await()
 
     verify(fileRepositoryMock).addFiles(expectedRow)
     captor.getAllValues.size should equal(1)
-    captor.getAllValues.get(0).head.consignmentid should equal(consignmentId)
-    captor.getAllValues.get(0).head.userid should equal(userId.toString)
+    captor.getAllValues.get(0).head.consignmentid should equal(consignmentUuid)
+    captor.getAllValues.get(0).head.userid should equal(userId)
```
Comment (TomJKing) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala:83]: Unnecessary line break

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/SeriesServiceSpec.scala [hunk 2]
```diff
@@ -15,17 +18,19 @@ class SeriesServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers {
   implicit val executionContext: ExecutionContext = ExecutionContext.Implicits.global
 
   "getSeries" should "return the specfic series for a body if one is provided" in {
+    val fixedUuidSource = new FixedUUIDSource()
     val repoMock = setupSeriesResponses
 
-    val seriesService: SeriesService = new SeriesService(repoMock)
+
```
Comment (TomJKing) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/service/SeriesServiceSpec.scala:24]: Unnecessary line break

File: src/test/scala/uk/gov/nationalarchives/tdr/api/utils/FixedUUIDMultipleSource.scala [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import uk.gov.nationalarchives.tdr.api.service.UUIDSource
+
+object FixedUUIDMultipleSource extends UUIDSource {
+  var idx = -1
+  override def uuid: UUID = {
+    idx += 1
+    List(
+    UUID.fromString("bbcae38a-4334-41a5-9564-99ad90b7254c"),
+    UUID.fromString("bbcae38a-4334-41a5-9564-99ad90b7254c"),
+    UUID.fromString("bbcae38a-4334-41a5-9564-99ad90b7254c")
```
Comment (suzannehamilton) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/utils/FixedUUIDMultipleSource.scala:14]: Are these supposed to be different UUIDs? Or has this class been replaced by the next one?

**Comment by suzannehamilton** [comment]
Could you update the README as well?

It's probably worth making sure everyone in the team's aware when you deploy this, because we'll all need to update our dev environments, and integration might be broken for a bit while the API and frontend are out of sync.

File: src/test/scala/uk/gov/nationalarchives/tdr/api/utils/FixedUUIDMultipleSource.scala (hunk 3 above)
Comment (MancunianSam) [reply src/test/scala/uk/gov/nationalarchives/tdr/api/utils/FixedUUIDMultipleSource.scala]: Good spot, this is unused now, I've deleted it.

**Comment by MancunianSam** [comment]
I've updated the README to say that you'll need aws credentials to build it. I've managed to get all the libraries in sync and deployed so once all of this code is merged, we should be fine.

File: src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala (hunk 1 above)
Comment (MancunianSam) [reply src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala:83]: I've removed this one and the one below.

**Pull request merged**